Dockerfile

```diff
@@ -24,5 +24,8 @@ ENV LC_ALL C.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US.UTF-8
 
+RUN echo "Testing program is on PATH"
+RUN bash --version
```

Comment:
I encountered an issue here where Codespaces was trying to install some additional things it needed inside my container, and on Alpine it didn't have `bash` installed by default, nor did it have `/bin` on the PATH.

This is a check to ensure we provide it for this tooling.